src/shapes/object.class.js

```diff
@@ -1113,7 +1113,7 @@
      * @returns Boolean
      */
     hasStroke: function() {
-      return this.stroke && this.stroke !== 'transparent' && this.strokeWidth !== 0;
+      return this.stroke && this.stroke !== 'transparent' && this.stroke !== 'none' && this.strokeWidth !== 0;
```

Comment:
none is not an accepted color and shouldn't be. none renders black has any string that is not a color and can't be parsed

Reply:
I removed it